src/components/Dashboard.vue

```diff
@@ -177,9 +177,20 @@
           data-cy="expenses"
         >
         </AdjustCounter>
-        <InfoButton
-          link="https://www.cgd.pt/Site/Saldo-Positivo/leis-e-impostos/Pages/deducoes-especificas.aspx#:~:text=Empresariais%20e%20Profissionais%20(Categoria%20B)&text=Se%20estiver%20enquadrado%20no%20regime,bruto%20(antes%20dos%20descontos)."
-        />
+        <span>
+          <InfoButton
+            link="https://www.cgd.pt/Site/Saldo-Positivo/leis-e-impostos/Pages/deducoes-especificas.aspx#:~:text=Empresariais%20e%20Profissionais%20(Categoria%20B)&text=Se%20estiver%20enquadrado%20no%20regime,bruto%20(antes%20dos%20descontos)."
+          />
+          <Transition>
+            <button
+              class="text-xs text-neutral-500"
```

Comment:
this could be horizontally aligned:

```html
<button
class="text-xs text-neutral-500 flex flex-col items-center"
```

or whatever is the simpler solution